spec/fun_ruby/modules/array_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
```

Comment:
@Kirillpt RSpec tests are not required, because the main idea is that the examples in the comments are tests themselves. You can run them using `bin/yard doctest`. So, you are free to delete this file.

Reply:
Okay, got it.